compiler/rustc_codegen_llvm/src/gotoc/mod.rs

```diff
@@ -268,8 +298,8 @@ impl CodegenBackend for GotocCodegenBackend {
                     MonoItem::Fn(instance) => c.declare_function(instance),
                     MonoItem::Static(def_id) => c.declare_static(def_id, item),
                     MonoItem::GlobalAsm(_) => {
```

Comment:
Not sure if there is a good way to handle this.

Reply:
We could at least give a warning, otherwise this will be hard to debug in the future.

Reply:
Done